packages/twenty-front/src/modules/ui/feedback/snack-bar-manager/hooks/useSnackBar.ts

```diff
@@ -54,16 +58,91 @@ export const useSnackBar = () => {
     [scopeId],
   );
 
-  const enqueueSnackBar = useCallback(
-    (message: string, options?: Omit<SnackBarOptions, 'message' | 'id'>) => {
+  const enqueueSuccessSnackBar = useCallback(
```

Comment:
question: not against it but not sure to see the big value of having created these specific functions over passing a variant as parameter